compose-builder/get-api-gateway-token.sh

```diff
@@ -24,16 +24,30 @@ if [ "$DEV" = "-dev" ]; then
   CORE_EDGEX_VERSION=0.0.0
 fi
 
-rm -rf keys
-mkdir keys
-openssl ecparam -name prime256v1 -genkey -noout -out keys/ec256.key 2> /dev/null
-cat keys/ec256.key | openssl ec -out keys/ec256.pub 2> /dev/null
+# Init key dir
+GW_KEY_DIR=/tmp/edgex-gateway-keys
+
+rm -rf ${GW_KEY_DIR}
+mkdir -p ${GW_KEY_DIR}
+
+# Build Gateway Key
+openssl ecparam -name prime256v1 -genkey -noout -out ${GW_KEY_DIR}/gateway.key 2> /dev/null
+openssl ec -in ${GW_KEY_DIR}/gateway.key -pubout -out ${GW_KEY_DIR}/gateway.pub 2> /dev/null
+
+# JWT File
+JWT_FILE=/tmp/edgex/secrets/edgex-security-proxy-setup/kong-admin-jwt
+JWT_VOLUME=/tmp/edgex/secrets/edgex-security-proxy-setup
 
 ID=`uuidgen`
-docker run --rm -it -e KONGURL_SERVER=kong --network edgex_edgex-network --entrypoint "" -v ${PWD}/keys:/keys \
+
+docker run --rm -it -e KONGURL_SERVER=kong --network edgex_edgex-network --entrypoint "" -v ${GW_KEY_DIR}:/keys -v ${JWT_VOLUME}:${JWT_VOLUME} \
```

Comment:
Would the volume mapping from host system of `$JWT_VOLUME` pose an issue if running on windows or non-linux?
suggest using $PWD on the host system side to avoid this potential issue.

Reply:
Given that it's a shell script, I don't see why it would be run in a non-linux shell?

Reply:
i guess i was thinking about `/tmp` may not be even available originally on non-linux but I do see now we create if not existing so i think it should be fine.   so i am fine with it now